test/integration/features/flags.feature

```diff
@@ -0,0 +1,51 @@
+@flags
+Feature: Flags
+  As a user I can perform advanced operations
+  such as starting Minishift with optional features
+
+  Scenario: Set configuration options for Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift config set docker-opt dns=8.8.8.8" succeeds
+     Then executing "minishift config get docker-opt" succeeds
```

Comment:
I think this is more of a 'And' - When I can set the config and and I get retrieve then, then ...